api/rpm_and_image_fetcher.py

```diff
@@ -0,0 +1,72 @@
+import os
+import requests
+import logging
+import json
+
+# Set your GitHub access token as an environment variable
+GITHUB_TOKEN = os.environ.get("GITHUB_PERSONAL_ACCESS_TOKEN")
+headers = {
+    "Authorization": f"token {GITHUB_TOKEN}"
+}
+
+REPO_OWNER = "openshift-eng"
+REPO_NAME = "ocp-build-data"
+
+def get_directory_contents(branch, directory):
+    path = f"{REPO_OWNER}/{REPO_NAME}/contents/{directory}?ref={branch}"
+    contents_api = f"https://api.github.com/repos/{path}"
+    response = requests.get(contents_api, headers=headers)
+    response.raise_for_status()  # Raise an exception if the request fails
+    return response
+
+def fetch_data():
+    result = []
+
+    # Separate entry for the "openshift-3.11" branch
+    branch_name = "openshift-3.11"
+    logging.debug(f"Processing branch {branch_name}...")
```

Comment:
Hmm can we log without using a logger though? Can you see the logs? I suspect that we need to create a logger `logger = logging.getLogger()` and then use `logger.info()`